### DISCUSSION TITLE
Is the seventh value of a color array in a Tuple the color used in named colors?

### DISCUSSION THREAD

**Discussion by BartAlcorn** [mantinedev/mantine#1326] [Q&A] [answered]
If I set the `primaryColor: 'brand'` in a MantineProvider theme objects the resulting primary color is the 7th ([6] because of zero indexing) color of the 'brand' array in the Tuple.

I am I correct this is the default behavior?
If this documented anywhere?
Has this changed recently? I was fairly certain until today that the default value was the fifth ([4]) value.

Thanks!

**Comment by rtivital** [comment, marked as answer]
- yes the index of primary shade is 6
- no it was not changed since 1.0 release
- 4.2 release will add an option to customize primary shade with property on theme

**Reply by BartAlcorn** [reply]
Thanks. Is there a rough roadmap time wise to v4.2?
